Atomic/atomic.py

```diff
@@ -645,36 +645,62 @@ def gen_cmd(self, cargs):
             args.append(c)
         return " ".join(args)
 
+    def get_fq_name(self, image_info):
+        if len(image_info['RepoTags']) > 1:
+            raise ValueError("\n{} is tagged with multiple repositories. "
+                             "Please use a repository name instead.\n".format(self.image))
+        else:
```

Comment:
I added a debug statement to show you what the return value is.  Hopefully that will clear things up for you.  Note the lines below that start with 'fq_name is...'

```
[bbaude@localhost atomic (skopeo_qualified)]$ sudo ./atomic info busybox
fq_name is docker.io/busybox:latest
'busybox' has no label information.

[bbaude@localhost atomic (skopeo_qualified)]$ sudo ./atomic info fedora
fq_name is docker.io/fedora:latest
'fedora' has no label information.

[bbaude@localhost atomic (skopeo_qualified)]$ sudo ./atomic info rsyslog
fq_name is registry.access.redhat.com/rhel7/rsyslog:latest
build-date: 2016-02-11T13:39:59.109718Z
RUN: docker run -d --privileged --name NAME --net=host --pid=host -v /etc/pki/rsyslog:/etc/pki/rsyslog -v /etc/rsyslog.conf:/etc/rsyslog.conf -v /etc/sysconfig/rsyslog:/etc/sysconfig/rsyslog -v /etc/rsyslog.d:/etc/rsyslog.d -v /var/log:/var/log -v /var/lib/rsyslog:/var/lib/rsyslog -v /run:/run -v /etc/machine-id:/etc/machine-id -v /etc/localtime:/etc/localtime -e IMAGE=IMAGE -e NAME=NAME --restart=always IMAGE /bin/rsyslog.sh
Name: rhel7/rsyslog
Build_Host: rcm-img-docker01.build.eng.bos.redhat.com
architecture: x86_64
vcs-type: git
vcs-ref: cc44dd261743db4185f2a1435e79ce607263f223
Version: 7.2
Architecture: x86_64
INSTALL: docker run --rm --privileged -v /:/host -e HOST=/host -e IMAGE=IMAGE -e NAME=NAME IMAGE /bin/install.sh
Release: 15
Vendor: Red Hat, Inc.
BZComponent: rsyslog-docker
Authoritative_Registry: registry.access.redhat.com
UNINSTALL: docker run --rm --privileged -v /:/host -e HOST=/host -e IMAGE=IMAGE -e NAME=NAME IMAGE /bin/uninstall.sh

```